### ISSUE TITLE
I don't quite understand the specific functions of this plugin

### ISSUE THREAD

**Issue by xuegodot** [S83gamedev/AudioStream-Subtitle-Plugin#1]
Can you record a usage video thanks

**Comment by S83gamedev** [comment]
> Can you record a usage video thanks

What part you don't understand?
just import the plugin - then go and add a node - then search for the audioSub - then go and put an audio file into the node and then write something on desc and then make sure to check autoplay on , then when you run the scene the audio plays and the subtitle will be shown too

TLDR: This customized node helps you to add subtitle more easily into the game

**Comment by S83gamedev** [comment]
> Can you record a usage video thanks

If you had any other question just ask me :D

**Comment by S83gamedev** [comment]
> Can you record a usage video thanks

Oh yeah I was planing to, hopefully I will do it one day 
___
yeah sorry at first I just saw the title and didn't read the description but now i did lol

**Comment by xuegodot** [comment]
> > 你能录制一个使用视频吗？
> 
> 你不明白什么？ 只需导入插件-然后去添加一个节点-然后搜索audioSub -然后去把一个音频文件到节点，然后在desc上写一些东西，然后确保检查自动播放，然后当你运行场景时，音频播放和字幕也会显示
> 
> TLDR：这个自定义的节点可以帮助您更轻松地将字幕添加到游戏中



> > Can you record a usage video thanks
> 
> What part you don't understand? just import the plugin - then go and add a node - then search for the audioSub - then go and put an audio file into the node and then write something on desc and then make sure to check autoplay on , then when you run the scene the audio plays and the subtitle will be shown too
> 
> TLDR: This customized node helps you to add subtitle more easily into the game

I initially thought it would be like displaying lyrics, with subtitles appearing in rhythm with the music. I even felt that I might have used it incorrectly. After watching your video, it became clear that I had misunderstood. However, this plugin is still very practical. Thank you.

**Comment by xuegodot** [comment]
> > Can you record a usage video thanks
> 
> What part you don't understand? just import the plugin - then go and add a node - then search for the audioSub - then go and put an audio file into the node and then write something on desc and then make sure to check autoplay on , then when you run the scene the audio plays and the subtitle will be shown too
> 
> TLDR: This customized node helps you to add subtitle more easily into the game

I initially thought it would be like displaying lyrics, with subtitles appearing in rhythm with the music. I even felt that I might have used it incorrectly. After watching your video, it became clear that I had misunderstood. However, this plugin is still very practical. Thank you.

**Comment by S83gamedev** [comment]
> I initially thought it would be like displaying lyrics, with subtitles appearing in rhythm with the music. I even felt that I might have used it incorrectly

Actually you can still do that, but it's not usual you can do that with even just a label node
so I will explain how to do that incase you are curious
1- create a label node for each time part of the lyric gonna show up on the screen
2- use animation player or give it a timer (my node already have a timer, but label node doesn't) (using animation player for when lable node gonna be visible or invisible, and timer for making it invisible automaticly, how ever you like, you can use each one you prefer or combining them)
<3
good luck

**Comment by S83gamedev** [comment]
> However, this plugin is still very practical. Thank you.

again sorry for late response tho :D
im kinda lazy and not in a very good mood and mental, but still managed to record the video
i'm glad it helped
